src/contracts/deploymentHelper/DeploymentHelper.sol

```diff
@@ -0,0 +1,63 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+pragma solidity ^0.8.10;
+
+import "../vendored/interfaces/MultiTokenMediatorInterface.sol";
+import "../CowProtocolVirtualToken.sol";
+
+/// @dev Contract contains the logic for deploying CowProtocolVirtualToken on gnosis chain
+/// @author CoW Protocol Developers
+contract DeploymentHelper {
```

Comment:
Out of personal preference, I'd call this `BridgedTokenDeployer`.
This is because `DeploymentHelper` is too generic, for example I'm also building right now a contract that could count as a deployment helper.